omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-SummaryPage.xml

```diff
@@ -0,0 +1,1732 @@
+                });
+
+                jq('[class^="ARTStartEnableDisable"]').change(function () {
+                    enable_disable_art_start(jq(this));
+                });
+
+
+                jq('[class^="ARTStartEnableDisable"]').each(function () {
+                    enable_disable_art_start(jq(this))
+                });
+
+
+                jq('[class^="ARTStart3EnableDisable"]').change(function () {
+                    enable_disable_art_start3(jq(this));
+                });
+
+
+                jq('[class^="ARTStart3EnableDisable"]').each(function () {
+                    enable_disable_art_start3(jq(this))
+                });
+
+
+                jq('[class^="ARTTreatmentInterruptionEnableDisable"]').change(function () {
+                    enable_disable_ati(jq(this));
+                });
+
+
+                jq('[class^="ARTTreatmentInterruptionEnableDisable"]').each(function () {
+                    enable_disable_ati(jq(this))
+                });
+
+                jq('[class^="PriorYesNoEnableDisable"]').change(function () {
+                    enable_disable(jq(this), "PriorYesNo", 'selected_value == "yes"', "select");
+                });
+
+                jq('[class^="PriorYesNoEnableDisable"]').each(function () {
+                    enable_disable(jq(this), "PriorYesNo", 'selected_value == "yes"', "select");
+                });
+
+                jq("#165143").addClass('hidden');
+
+                jq(".enableDisable").each(function () {
+                    var group = jq(this);
+
+                    function disableFn() {
+                        group.children("#disabled").fadeTo(250, 0.25);
+                        /*  group.children("#disabled").find(":checkbox").attr("checked", false); //uncheck*/
+                        group.children("#disabled").find("input").attr("disabled", true);
+                        /*disable*/
+                        group.children("#disabled").find("select").attr("disabled", true);
+                    }
+
+                    function enableFn() {
+                        group.children("#disabled").fadeTo(250, 1);
+                        group.children("#disabled").find("input").attr("disabled", false);
+                        group.children("#disabled").find("select").attr("disabled", false);
+
+                    }
+
+                    if (jq(this).children("#trigger").find(":checkbox:first").attr("checked") != "checked") {
+                        disableFn();
+                    }
+
+                    jq(this).children("#trigger").find(":checkbox:first").change(function () {
+                        var checked = jq(this).attr("checked");
+                        if (checked == "checked") {
+                            enableFn();
+                        } else {
+                            disableFn();
+                        }
+                    });
+                });
+
+
+                jq(".checkboxGroup").each(function () {
```

Comment:
@dbaluku this function should be generalized in the poc.js file

Reply:
Done